pkgs/by-name/gi/gitlab-container-registry/package.nix

```
@@ -35,13 +35,7 @@ buildGoModule rec {
   meta = with lib; {
     description = "GitLab Docker toolset to pack, ship, store, and deliver content";
     license = licenses.asl20;
-    maintainers =
-      with maintainers;
-      [
-        leona
-        yayayayaka
-      ]
-      ++ teams.cyberus.members;
+    teams = with teams; [ gitlab cyberius ];
```

Eval error for some reason? Even though 
 we are following https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#teams-var-meta-teams ?

---

```suggestion
    teams = with teams; [ gitlab cyberus ];
```

Right that should fix it